tableauserverclient/models/personal_access_token_auth.py

```diff
@@ -0,0 +1,10 @@
+class PersonalAccessTokenAuth(object):
+    def __init__(self, token_name, personal_access_token, site_id=''):
+        self.token_name = token_name
+        self.personal_access_token = personal_access_token
+        self.site_id = site_id
+        # Personal Access Tokens doesn't support impersonation.
+        self.user_id_to_impersonate = None
```

Comment:
Why include it in the object if it's not supported?

Reply:
Sign in logic will check against it to see if we are trying to impersonate and if it should be added in the sign in request.